js/exercise.js

```diff
@@ -4,6 +4,12 @@ function loadExercise() {
     var exerciseReal = JSON.parse(localStorage.getItem(exerciseId));
 
     document.getElementById("exerciseName").innerHTML = exerciseReal.primaryName;
+    
+    var executed = "";
+
+    if (exerciseReal.workoutExercises[0].executed == true) {
```

Comment:
"== true" nao eh necessario se for verificar que eh verdadeiro, o if ja espera que a variavel ou sentenca seja verdadeira, caso contrario, falha